src/main/kotlin/io/zerobase/smarttracing/Main.kt

```diff
@@ -18,9 +19,10 @@ import javax.servlet.DispatcherType
 import javax.servlet.FilterRegistration
 
 typealias MultiMap<K,V> = Map<K, List<V>>
+typealias EmailConfig = Map<EmailType, EmailConfigEntry>
```

Comment:
you can import this type alias from the models file instead of redefining it

Reply:
my bad this was supposed to be deleted from the pushed code